src/main.ts

```diff
@@ -160,8 +161,10 @@ async function run(): Promise<void> {
       exit 2;
     fi
 
+    target=$(echo $run | cut -d'/' -f2)
+
     if [ -n "${coverageOutput}" ]; then
-      ${cli} --verbosity ${verbosity} download  --owner ${owner} $run -o ${coverageOutput};
+      ${cli} --verbosity ${verbosity} download  --owner ${owner} ${project}/$target -o ${coverageOutput};
```

Comment:
$target -> ${target} for consistency

Reply:
target is a bash variable not a javascript one like project that's why I couldn't add the `{}`